internal/pkg/cli/app_init.go

```diff
@@ -53,11 +53,40 @@ type initAppOpts struct {
 	prompt               prompter
 	prog                 progress
 	iam                  policyLister
+	iamRoleManager       roleManager
 	isSessionFromEnvVars func() (bool, error)
 
 	cachedHostedZoneID string
 }
 
+type errAppAlreadyExistsInAccount struct {
```

Comment:
Can we move these errors to near the bottom of the file, just above `func buildAppInitCommand() *cobra.Command {`, i.e. around L325?